code/game/turfs/simulated/walls_mineral.dm

```diff
@@ -28,10 +28,8 @@
 
 /turf/simulated/wall/mineral/wood/ex_act(var/severity)
 	if(severity < 3)
-		ChangeTurf(get_underlying_turf())
```

Comment:
Shouldn't you look at the reason this call doesn't work the way you want rather than ripping it out?